Dockerfile

```diff
@@ -9,18 +9,15 @@ RUN apt-get -yqq update
 
 WORKDIR /root/bot
 
-ADD https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/python_mysql.py \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slack_cmd_process.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slack_message.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/slackbot.py  \
- https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/start_app.py /root/bot/
+COPY python_mysql.py slack_cmd_process.py slack_message.py slackbot.py start_app.py /root/bot/
 
-ADD https://raw.githubusercontent.com/USDevOps/jenkins_slack_bot/tag_v1/init.sql /docker-entrypoint-initdb.d/
+COPY init.sql /docker-entrypoint-initdb.d/
 
 RUN chmod 775 /docker-entrypoint-initdb.d/init.sql && chmod -R 775 /root
```

Comment:
Let's place everything to the working directory?

Instead of /root/ and /

Also chmod can only happen for that working directory

Reply:
Init.sql has to be put in only /docker-entrypoint-initdb.d/. MySQL image default entry point run this SQL file during initialization

Reply:
@vivekgrover1 , np

And let's change to "chmod -R 775 /root" to "chmod -R 755 /root/bot/"?

Reply:
@DennyZhang yes that is fine.